iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/TieLineWithAliasesXmlTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.iidm.xml;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * @author Sebastien Murgey <sebastien.murgey@rte-france.com>
+ */
+public class TieLineWithAliasesXmlTest extends AbstractXmlConverterTest {
+
+    @Test
+    public void test() throws IOException {
+        roundTripVersionedXmlTest("tielineWithAliases.xml", IidmXmlConstants.CURRENT_IIDM_XML_VERSION);
+
+        // Tests for backward compatibility
+        roundTripVersionedXmlFromMinToCurrentVersionTest("tielineWithAliases.xml", IidmXmlVersion.V_1_3);
```

Comment:
Both are redundant?

Reply:
Not in next version of IIDM

Reply:
You right, the current_version is ignored. I ask to @MioRtia if the expected behavior